vendor/github.com/hpcloud/tail/tail.go

```diff
@@ -158,6 +158,17 @@ func (tail *Tail) Tell() (offset int64, err error) {
 	return
 }
 
+// Size returns the length in bytes of the file being tailed,
+// or 0 with an error if there was an error Stat'ing the file.
+func (tail *Tail) Size() (size int64, err error) {
+	fi, err := tail.file.Stat()
```

Comment:
scope of err is confusing.

I think we should removed the named returns they are not helpful.

Reply:
yeah, i don't disagree, I only did it this way to be consistent with the way the rest of this library is written

Reply:
I see but I still would change it, not clear if this work correctly as `err` scope is ambiguous.